apel/db/unloader.py

```diff
@@ -72,6 +91,7 @@ def _get_base_query(self, record_type):
         query = Query()
 
         if record_type in (JobRecord, SummaryRecord, NormalisedSummaryRecord,
+                           JobRecord04, SummaryRecord04, NormalisedSummaryRecord04,
```

Comment:
Just a clarification: 

Say, Before if was a like JobRecord, NormalisedSummaryRecord we were doing the checks like whether they are having include_vos, exclude_vos or local_jobs etc..

Since we have added new record types(like JobRecord04, NormalisedSummaryRecord04)  Are we NOT doing the check anymore for the previous record types?